addons/crm/models/crm_lead.py

```diff
@@ -1807,6 +1807,13 @@ def _message_get_suggested_recipients(self):
             pass
         return recipients
 
+    def _message_merging_leads_failed(self, body):
+        subject = _('Merging Leads Failed')
+        self[0].message_post(body=body, subject=subject)
+        for lead in self[1:]:
+            lead._message_log(body=body, subject=subject)
+        return
```

Comment:
fyi, I came accross `_merge_log_summary` recently, and thought about this task ^^
Maybe just logging it in the summary at the end would be better, maybe to see with lna.